### ISSUE TITLE
Capture the ColumnResize event

### ISSUE THREAD

**Issue by NageshAndani** [material-table-core/core#400]
Whenever user resizes the column of a table, I would like to capture the columnResize event and store the new width in the browser's local storage, so that, on reopening the page, user can restore his column width applied.

However, I don't see any event exposed by this Table component to capture the columnResize event.
Can you help me understand how to achieve this?

**Comment by Domino987** [comment]
You would need to add a prop to be called on the resize event form within the code I think. You could try to add it and create a PR.

**Comment by matthewoestreich** [comment]
@NageshAndani please let me know if you need some help with this! It looks like you'll just need to pass that event up the chain and expose it. I will reopen this if needed. Thanks for your contribution!

**Comment by jet11x** [comment]
I've been looking into this and other aspects of column resizing.  I've got changes working locally that:
- allow table to expand as column are resized
- added onColumnResize callback
- supress resize icon when table header cells have borders

I've got some tidying up to do then I'll submit a pull for consideration.

I'd already made one local patch to material-table and then found another bug.  Was lucky to come across material-table-core - nice piece of work.
